docusaurus/docs/get-started/get-started.mdx

```diff
@@ -15,6 +15,28 @@ import ScaffoldNPM from '@snippets/createplugin-scaffold.npm.md';
 import ScaffoldPNPM from '@snippets/createplugin-scaffold.pnpm.md';
 import ScaffoldYarn from '@snippets/createplugin-scaffold.yarn.md';
 
+# Grafana plugin developer's guide
+
+Extend Grafana's built-in capabilities with plugins. Plugins enable Grafana 
+to accomplish specialized tasks, custom-tailored to your requirements. 
+By making a plugin for your organization, you can connect Grafana to other 
+data sources, ticketing tools, and CI/CD tooling.
+
+You can create plugins for private use or contribute them to the open 
+source community by publishing to the Grafana 
+[plugin catalog](https://grafana.com/plugins). 
+This catalog has hundreds of other community and commercial plugins.
+
+## Types of plugins
```

Comment:
Unfortunately adding this here introduces duplication with https://grafana.com/developers/plugin-tools#what-type-of-plugin-would-you-like slightly further down. Maybe that's fine, but feels a bit strange. Also we should ensure the descriptions match.

Reply:
Agreed, removed.